homeassistant/components/kermi/config_flow.py

```diff
@@ -0,0 +1,52 @@
+"""Config flow for kermi."""
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry, ConfigFlowResult
+from homeassistant.const import CONF_HOST
+
+from .const import DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class KermiConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Kermi."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    async def async_setup_entry(self, entry: ConfigEntry) -> bool:
+        """Set up a config entry."""
+        # Set the unique ID of the config flow to the entry_id of the config entry
+        await self.async_set_unique_id(entry.entry_id)
+        return True
+
+    async def async_step_user(self, user_input=None) -> ConfigFlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        if user_input is not None:
+            # If the input is valid, create the config entry
+            if not errors:
+                entry = self.async_create_entry(
+                    title=user_input[CONF_HOST],
+                    data=user_input,
+                )
+                self._abort_if_unique_id_configured()
```

Comment:
if you really need to, do the `await self.async_set_unique_id(entry.entry_id)` here, but is there no other option for a unique id?